all/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+# This is the lists file that should be used during development for the best
+# discovery in the developer's choice of IDE.
+cmake_minimum_required(VERSION 3.2)
+
+project(FunctionalPlusDevelopment)
+
+# Test should be added first, because it disables the warning guard
+add_subdirectory(
+    "${PROJECT_SOURCE_DIR}/../test"
+    "${PROJECT_BINARY_DIR}/test"
+)
+
+# This project supports CMake 3.2 as a minimum version, so there is no way to
+# protect against multiple inclusions other than to turn add_subdirectory into
+# a noop with code injection at project() calls.
+# If at least CMake 3.11 were to be supported, then the FetchContent module has
```

Comment:
Ubuntu Bionic ships 3.10 (:cry:) but with Focal it's 3.16 (:+1:).

Reply:
3.11. would mean, it would work with Disco Dingo (having 3.12) and up out of the box. For earlier Ubuntu versions one would need to upgrade, which is not too hard with a PPA. So I'd say, let's go with 3.11. (and adjusting [`.travis.yml`](https://github.com/Dobiasd/FunctionalPlus/blob/ec29ec81ab42920967f530a561ef280c1e768a8a/.travis.yml#L3) to `dist: focal`) if this allows implementing things cleaner, and you two don't see a big issue with that. :+1:

Reply:
Sounds good to me!